### PR TITLE
MCH: improved naming and location of digits task histograms

### PR DESCRIPTION
**Pull request by aferrero2707**
The DE-specific occupancy histograms are moved into sub-folders, following the convention "STx/DExxx/".
This allows to only have global histograms in the root "QcTaskMCHDigits" folder.
Histogram names have also been shortened for better readability.

### CODE REVIEW

**Comment by Barthelemy** [comment]
@TimoWilken @ktf Macos build failed without logs

File: Modules/MUON/MCH/src/PhysicsTaskDigits.cxx [hunk 1]
```diff
@@ -49,6 +49,11 @@ PhysicsTaskDigits::PhysicsTaskDigits() : TaskInterface() {}
 
 PhysicsTaskDigits::~PhysicsTaskDigits() {}
 
+static std::string getHistoPath(int deId)
+{
+  return fmt::format("ST{}/DE{}/", (deId - 100) / 200 + 1, deId);
```
Comment (Barthelemy) [inline Modules/MUON/MCH/src/PhysicsTaskDigits.cxx:54]: magic numbers ? :)

**Review by Barthelemy — APPROVED** [review]

**Comment by TimoWilken** [comment]
> Macos build failed without logs

It was a temporary failure fetching a dependency, I've restarted the check.

File: Modules/MUON/MCH/src/PhysicsTaskDigits.cxx (hunk 1 above)
Comment (aferrero2707) [reply Modules/MUON/MCH/src/PhysicsTaskDigits.cxx:54]: Yes, but graved in the stone ;-)

**Pull request merged**